compiler-plugin-tests/src/test/java/io/ballerina/stdlib/ftp/plugin/FTPServiceValidationTest.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.ftp.plugin;
+
+import io.ballerina.projects.DiagnosticResult;
+import io.ballerina.projects.Package;
+import io.ballerina.projects.PackageCompilation;
+import io.ballerina.tools.diagnostics.Diagnostic;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static io.ballerina.stdlib.ftp.plugin.CompilerPluginTestUtils.assertDiagnostic;
+import static io.ballerina.stdlib.ftp.plugin.CompilerPluginTestUtils.loadPackage;
+import static io.ballerina.stdlib.ftp.plugin.PluginConstants.CompilationErrors;
+
+/**
+ * Tests for FTP package compiler plugin.
+ */
+public class FTPServiceValidationTest {
```

Comment:
Shall we rename this to `FtpServiceValidationTest`? Please fix in other places as well.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-ftp/pull/558/commits/2ab5e744a42a7f4a71616b855ca16ef94533eb62